monero-outreach-docs/translations/nl/PSA/files/wallet.md

```diff
@@ -0,0 +1,26 @@
+# Een veilige wallet is een gelukkige wallet
```

Comment:
### Enkele aanbevolen methodes omtrent walletveiligheid:

1. Om te voorkomen dat oplichters je XMR stelen, gebruik je uitsluitend wallets goedgekeurd door de Monero-gemeenschap. Doe je research vooraleer je eender welke wallet opzet. Een goed begin is de informatiesectie van r/Monero (https://www.reddit.com/r/Monero/).  

2. Als je een cold wallet (bijvoorbeeld een "paper" wallet) voor langdurige opslag aanmaakt, doe dan ook je research, en volg een online gids met de meest overtuigende en positieve recensies en commentaar. Zoals deze links van Reddit:

  a. Gids voor het zenden naar een paper wallet
r/Monero/comments/7t5tyx/tutorial_sending_xmr_to_a_paper_wallet_generated

  b. Gids voor het veilig aanmaken van een cold wallet
r/Monero/comments/48cgmd/an_extensive_guide_for_securely_generating_an

3. Overweeg een cold wallet voor de XMR die je plant bij te houden, en een hot wallet voor XMR die je wil spenderen of verhandelen. Cold wallet = spaarrekening; hot wallet = zichtrekening.

4. Bezoek online fora, vraag rond, en leer de verschillende manieren waarop privésleutels en herstelteksten veilig op te bergen, en kies de beste manier voor jouw situatie. Begin hier: https://getmonero.org/community/hangouts/. Elk plan is beter dan geen plan. Heb je al door dat research een belangrijk thema is?

5. Volg andere aanbevolen methodes zoals nooit je privésleutels delen, enkel reputabele beurzen gebruiken, je wallet(s) updaten na elke netwerkupgrade of hard fork, je balans verzwijgen, en altijd leergierig te zijn!